kraken/kubernetes/client.py

```diff
@@ -15,10 +15,13 @@ def initialize_clients(kubeconfig_path):
 
 
 # List nodes in the cluster
-def list_nodes():
+def list_nodes(label_selector=None):
     nodes = []
     try:
-        ret = cli.list_node(pretty=True)
+        if label_selector:
+            ret = cli.list_node(pretty=True, label_selector=label_selector)
```

Comment:
Since you are giving label_selector=None, does just using line 22 work instead of the if statement? 

Also, I do not see anywhere where you call list_nodes and pass it a label_selector. Is that correct?

Reply:
Yeah, previously in [line](https://github.com/openshift-scale/kraken/pull/25/files#diff-1da5ecddeb0574d357ad50edac640d84R15), list_nodes was used instead of list_killable_node.